manual/docker/Enterprise-Edition/Deploy SeaTable-EE with Docker.md

```diff
@@ -114,18 +121,6 @@ NOTE: The first command uses the option `-d` which starts the service in the bac
 
 You can now access SeaTable at the host name.
 
```

Comment:
Perhaps leave a short note here that since version 2.6 SeaTable EE won't start without a license.

Reply:
Modification made.